src/main/java/seedu/algobase/model/plan/Plan.java

```diff
@@ -101,15 +104,15 @@ public boolean equals(Object other) {
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, description, tasks, startDate, endDate);
+        return Objects.hash(planName, planDescription, startDate, endDate, tasks);
     }
 
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
-                .append(" Description: ")
-                .append(getDescription())
+        builder.append(getPlanName())
+                .append(" PlanDescription: ")
```

Comment:
Hmm this is for displaying purpose, so either `Description` or `Plan Description`?